### PR TITLE
Event calendar UI and selection of Time/Date of event

### PR DESCRIPTION
**Pull request by TheAmericanMomo**
In this pull request I implement both of my tasks.

Firstly you have the Time and Date selection of an event. This is done during the creation of an Event. You will find this change in the EventCreatorActivity. Also updated Events to have an associated date.

Secondly, I implemented the Calendar UI. Each user can open a list view on the events that he is registered to. This is implemented in the RegisteredEventsCalendarActivity. The profile firebase data source was updated for the storing and fetching of the Calendar Events.

### CODE REVIEW

File: app/src/main/java/com/sdpteam/connectout/profile/ProfileFirebaseDataSource.java
Comment (Lukovitch) [inline app/src/main/java/com/sdpteam/connectout/profile/ProfileFirebaseDataSource.java:1]: good commentary on this class

File: app/src/main/java/com/sdpteam/connectout/profile/Profile.java [hunk 1]
```diff
@@ -1,6 +1,30 @@
 package com.sdpteam.connectout.profile;
 
 public class Profile {
+
+    public static class CalendarEvent {//TODO maybe the error comes from this static
```
Comment (Lukovitch) [inline app/src/main/java/com/sdpteam/connectout/profile/Profile.java:5]: remove this TODO please

File: app/src/main/java/com/sdpteam/connectout/event/viewer/RegisteredEventsCalendarActivity.java
Comment (Lukovitch) [inline app/src/main/java/com/sdpteam/connectout/event/viewer/RegisteredEventsCalendarActivity.java:1]: Nice

File: app/src/main/java/com/sdpteam/connectout/event/EventFirebaseDataSource.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
 package com.sdpteam.connectout.event;
 
+import android.util.Log;
```
Comment (Lukovitch) [inline app/src/main/java/com/sdpteam/connectout/event/EventFirebaseDataSource.java:3]: Maybe this import is useless

**Review by Lukovitch — CHANGES_REQUESTED** [review]
NIce job, you are really good. Keep it up. Just, be more carefull with the import and the TODO to remove

File: app/src/main/java/com/sdpteam/connectout/event/EventFirebaseDataSource.java (hunk 2 above)
Comment (TheAmericanMomo) [reply app/src/main/java/com/sdpteam/connectout/event/EventFirebaseDataSource.java:3]: Indeed! Forgot to remove it after my debugging

**Review by Lukovitch — APPROVED** [review]
Thanks for the changes

**Pull request merged**